### PR TITLE
feat: 🎸 connect canister id

### PR DESCRIPTION
**Pull request by OyinloluB**
## Why?

Develop Connect Canister Id UI screen

## Contribution checklist?

- [x] The commit messages are detailed
- [x] It does not break existing features (unless required)
- [x] I have performed a self-review of my own code
- [ ] Documentation has been updated to reflect the changes
- [ ] Tests have been added or updated to reflect the changes
- [x] All code formatting pass
- [x] All lints pass

## Demo?

https://user-images.githubusercontent.com/51888121/185112150-96d10953-a0bd-42b2-8d25-2bf0ff6932b7.mov

Screenshot 2022-08-17 at 03 59 52

### CODE REVIEW

File: src/components/onboarding/connect-canister-id.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+  return (
+    <SectionWrapper>
+      <div>
+        <SectionTextContent>
+          <Header>
+            1. {t('translation:onboarding.connectCollection')}
+          </Header>
+          <SubText>
+            For now, we're providing the insertion of existing
```
Comment (prasanthLalapeta) [inline src/components/onboarding/connect-canister-id.tsx:24]: Let's move text to translations

File: src/components/core/buttons/styles.ts [hunk 2]
```diff
@@ -52,9 +52,9 @@ export const Button = styled('button', {
         border: '1.5px solid $primary',
       },
       active: {
-        background: 'rgba(34, 83, 255, 0.12)',
-        color: '$primary',
-        border: '1.5px solid $primary',
+        background: 'rgba(34, 83, 255, 0.16)',
+        color: '$tableLinkTextColor',
```
Comment (prasanthLalapeta) [inline src/components/core/buttons/styles.ts:56]: We can rename the color names and use it here. Colours should be named per colour for reusability, not functionality.

File: src/components/onboarding/connect-canister-id.tsx (hunk 1 above)
Comment (OyinloluB) [reply src/components/onboarding/connect-canister-id.tsx:24]: Yeah, I'll do this, still trying to figure out how to add the link tags to the translation file. Any ideas?

Comment (prasanthLalapeta) [reply src/components/onboarding/connect-canister-id.tsx:24]: For links it's better to break down the text into multiple values

Comment (OyinloluB) [reply src/components/onboarding/connect-canister-id.tsx:24]: Multiple values? Can you explain further?

**Review by prasanthLalapeta — APPROVED** [review]

**Pull request merged**